crates/matrix-sdk-crypto/src/olm/account.rs

```diff
@@ -352,6 +358,9 @@ pub struct PickledAccount {
     pub pickle: AccountPickle,
     /// Was the account shared.
     pub shared: bool,
+    /// Whether this is for a dehydrated device
+    #[serde(default)]
+    pub dehydrated: bool,
```

Comment:
adding this flag here will break things that create PickledAccount, such as https://github.com/matrix-org/matrix-rust-sdk-crypto-wasm/blob/main/src/libolm_migration.rs#L136  I'm not sure if there's a good way to deal with this.

Reply:
They'll need to update the bindings just like you did with the `matrix-sdk-crypto-ffi` bindings.

Reply:
OK, I just wasn't sure if there was a better way to do it.  I assume this means that the version will need to be bumped since this is a breaking change.  Is there a way to flag this so that when the next release gets made, the version will be bumped correctly?

Reply:
We're still in the `0.X.Y` range of releases, so the correct version will get bumped by design. Adding this as a breaking change to the changelog might make sense to notify people about it.